### ISSUE TITLE
Cannot get Call Screen Option 

### ISSUE THREAD

**Issue by Magchild** [jacopotediosi/GoogleDialerMod#68]
## Overview
[NOTE]: # ( Give a BRIEF summary about your problem )
Cannot get the option for "Spam and Call Screen".  Only have "Caller ID & spam"

## Steps to Reproduce
[NOTE]: # ( Provide a simple set of steps to reproduce this bug. )
1.  Google Dialer (Phone) app already systemized
2.  Forced closed google dialer multiple times
3.  Deleted app data and rebooted multiple times.

## Expected Behavior
[NOTE]: # ( Tell us what you expected to happen )

## Actual Behavior
[NOTE]: # ( Tell us what actually happens )

## Screenshots
[NOTE]: # ( If applicable, add screenshots to help explain your problem. )
![Screenshot_20230421-122802_Phone](https://user-images.githubusercontent.com/100013198/233688246-2c5c1c0e-2f91-409f-9f10-4c431b524de8.png)
![Screenshot_20230421-122814_GoogleDialerMod](https://user-images.githubusercontent.com/100013198/233688247-0cc72fb5-c31c-49c5-956d-fac51cfcc279.png)
![Screenshot_20230421-122819_GoogleDialerMod](https://user-images.githubusercontent.com/100013198/233688249-ea3f12bf-e1b8-44e1-8379-279916ae871f.png)

## System Information
- Device and model: 
Sony Xperia 5 II 

- ROM and Android version: 
LineageOS 20.0

- Is Google Dialer installed as system app: 
Yes (App Systemizer v17.4.1 Magisk Module)

- Installed Magisk / other SU Manager version: 
Magisk 26.1

[NOTE]: # ( Paste below the output of the `adb shell "dumpsys package com.jacopomii.googledialermod | grep version"` command )
- Installed GoogleDialerMod version:
 versionCode=200 minSdk=21 targetSdk=33
    versionName=2.00
    signatures=PackageSignatures{62465dd version:2, signatures:[8c539576], past signatures:[]}

[NOTE]: # ( Paste below the output of the `adb shell "dumpsys package com.google.android.dialer | grep version"` command )
- Installed Google Dialer version:
versionCode=11645988 minSdk=24 targetSdk=33
    versionName=103.0.522355693-publicbeta
    signatures=PackageSignatures{5ab1661 version:3, signatures:[788396df], past signatures:[]}
    versionCode=11645988 minSdk=24 targetSdk=33
    versionName=103.0.522355693-publicbeta
    signatures=PackageSignatures{ac73674 version:0, signatures:[], past signatures:[]}

[NOTE]: # ( Paste below the output of the `adb shell "getprop | grep locale"` command )
- Your device language (locale):
[persist.sys.locale]: [en-US]
[ro.product.locale]: [en-US]

[NOTE]: # ( Paste below the output of the `adb shell "getprop | grep iso-country"` command )
- Your location (country of the SIM and country where you are):
[gsm.operator.iso-country]: [us,]
[gsm.sim.operator.iso-country]: [us]

## Logcat
[NOTE]: # ( 
Launch the Dialer in Debug mode using the `adb shell "am start -D com.google.android.dialer"` command.
Open another terminal and use the `adb logcat > logs.txt` command to start capturing logs.
Perform the necessary steps to replicate the bug, then press CTRL+C to stop capturing logs.
Attach below the resulting logs.txt file.)
[logs.txt](https://github.com/jacopotediosi/GoogleDialerMod/files/11297523/logs.txt)

**Comment by jacopotediosi** [comment]
Can you please give a try with the latest debug version? You can download it from here: https://github.com/jacopotediosi/GoogleDialerMod/issues/51#issuecomment-1515212891.
Being a debug apk, you have to uninstall your current version to install that.

**Comment by jacopotediosi** [comment]
From your logcat: `04-21 12:12:35.665 29854 29854 W DialerRevertedFlagsDetector: com.android.dialer.compat.directboot.RevertedFlagsDetector.interacted:108 All flags are using default values for unknown reasons.`

This is very suspicious. You are on Lineage, so I suppose you are using Open GApps (?).
Can you check if Google Play Services is updated and if it is installed as a system app?

**Comment by Magchild** [comment]
I used MindTheGapps-13.0.0-arm64-20221025_100653 when setting up the phone.

Google Play Services seem up to date.
version 23.13.12 (190400-519946965)
com.google.android.gms

I believe it is already a system app because I cannot uninstall it from the settings>apps... menu.

**Comment by Magchild** [comment]
Yes, I have cleared storage and cleared cache, then rebooted, then force closed the dialer app multiple times.
